checkbox-provider-ce-oem/units/mir/jobs.pxu

```diff
@@ -0,0 +1,29 @@
+id: mir/check-ubuntu-frame-launching-auto
+category_id: mir_test
+plugin: shell
+flags: also-after-suspend
+user: root
+imports:
+    from com.canonical.certification import snap
```

Comment:
I am curious about this line is necessary or not, if we want to use the `sanp` in `requires` field ~~
Cos i still cna use the `snap.name` w/o add that ~~

Reply:
Not sure what was your environment. I just tested it again, without `from com.canonical.certification import snap` the job will not be generated.
```
# with importing snap
ceqa@ubuntu:~$checkbox.checkbox-cli list-bootstrapped com.canonical.qa.ceoem::ce-oem-mir-automated
Using sideloaded provider: checkbox-provider-ce-oem, version 0.1 from /var/tmp/checkbox-providers/checkbox-provider-ce-oem
Skipped file: /var/tmp/checkbox-providers/checkbox-provider-ce-oem/bin/socketcan_busoff_test.py
com.canonical.certification::snap
com.canonical.qa.ceoem::mir/check-ubuntu-frame-launching-auto
com.canonical.qa.ceoem::mir/glmark2-es2-wayland-auto
ceqa@ubuntu:~$

# without importing snap
ceqa@ubuntu:~$ checkbox.checkbox-cli list-bootstrapped com.canonical.qa.ceoem::ce-oem-mir-automated
Using sideloaded provider: checkbox-provider-ce-oem, version 0.1 from /var/tmp/checkbox-providers/checkbox-provider-ce-oem
Skipped file: /var/tmp/checkbox-providers/checkbox-provider-ce-oem/bin/socketcan_busoff_test.py
ceqa@ubuntu:~$